src/ZeekPluginConfig.cmake.in

```diff
@@ -0,0 +1,39 @@
+# Note: this config is used for builtin dynamic plugins outside of the source
+#       tree of Zeek. Plugins refer back to the Zeek source tree since they need
+#       access to some parts of Zeek that we don't install.
+
+# For finding zeek-plugin-create-package.sh and zeek-plugin-install-package.sh.
+set(ZEEK_PLUGIN_SCRIPTS_PATH "@ZEEK_PLUGIN_SCRIPTS_PATH@")
+
+# For finding Zeek sources.
+set(ZEEK_SOURCE_DIR "@ZEEK_SOURCE_DIR@")
+
+# Provide a hint to ZeekConfig.cmake where to find Broker from the build tree.
+# Note: the straightforward way would be setting `Broker_ROOT` instead, but
+#       plugins may still use CMake < 3.12.
+if (NOT "@ZEEK_HAS_EXTERNAL_BROKER@")
+  set(Broker_DIR "@ZEEK_PLUGIN_BROKER_PATH@" CACHE
+      PATH "Directory for finding Broker's package file" FORCE)
+endif ()
+
+# For having a binpac target available. Guarded to shield against including this
+# file multiple times.
+if (NOT TARGET Zeek::BinPAC)
+    add_executable(Zeek::BinPAC IMPORTED)
+    set_property(TARGET Zeek::BinPAC PROPERTY
+                 IMPORTED_LOCATION "@BINPAC_EXE_PATH@")
+endif ()
+
+# For having a bifcl target available. Guarded to shield against including this
+# file multiple times.
+if (NOT TARGET Zeek::BifCl)
+    add_executable(Zeek::BifCl IMPORTED)
+    set_property(TARGET Zeek::BifCl PROPERTY
+                 IMPORTED_LOCATION "@BIFCL_EXE_PATH@")
+endif ()
+
+# TODO: using BIFCL_EXE_PATH and BINPAC_EXE_PATH does not play well with
```

Comment:
I assume this can be follow-on work?

Reply:
Yep, I plan on doing that as a follow-up.